SDL/EndcapGeometry.cc

```diff
@@ -1,27 +1,16 @@
 #include "EndcapGeometry.h"
 
-SDL::EndcapGeometry* SDL::endcapGeometry = new SDL::EndcapGeometry();
-
-void SDL::freeEndcap() {
-  if (SDL::endcapGeometry != nullptr) {
-    delete SDL::endcapGeometry;
-    SDL::endcapGeometry = nullptr;
-  }
-}
-
-SDL::EndcapGeometry::EndcapGeometry(unsigned int sizef)
+SDL::EndcapGeometry<SDL::Dev>::EndcapGeometry(unsigned int sizef)
```

Comment:
if I understand correctly, `Acc` is more complex and combines both the memory representation (relevant for `Dev`) and the execution details.
I wanted to be minimal.

I had to move from `Acc` to `Dev` for buffers in the previous PR for compatibility with CMSSW. Here the dependence is only on buffers. If we decide to expose this part to ES, it will need to stay dependent on `DeV`